doc/manual/src/glossary.md

```diff
@@ -95,14 +103,12 @@
     then be built.
 
   - [reference]{#gloss-reference}\
-    A store path `P` is said to have a reference to a store path `Q` if
-    the store object at `P` contains the path `Q` somewhere. The
-    *references* of a store path are the set of store paths to which it
-    has a reference.
+    A [store path] `P` is said to have a *reference* to a store path `Q` if the string `Q` appears in the [store object] at `P`.
```

Comment:
```suggestion
    A [store object] `O` is said to have a *reference* to a store object `P` if the store path of `P` appears in the contents of`O`.
```

@Ericson2314 @edolstra does that sound about right?

Reply:
This sounds good to me!